extra_scripts/codemirror/theme/theboosters-dark.css

```
@@ -0,0 +1,122 @@
+  background: #16191d;
+  color: #e6edf3;
+}
+.cm-s-theboosters-dark div.CodeMirror-selected {
+  background: #2d4b6e;
+}
+.cm-s-theboosters-dark .CodeMirror-line::selection,
+.cm-s-theboosters-dark .CodeMirror-line > span::selection,
+.cm-s-theboosters-dark .CodeMirror-line > span > span::selection {
+  background: #2d4b6e;
+}
+.cm-s-theboosters-dark .CodeMirror-line::-moz-selection,
```

Stylelint (17.14.0)

[error] Vendor-prefixed selector "::-moz-selection" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)